swatch-product-configuration/src/main/java/com/redhat/swatch/configuration/model/Subscription.java

```diff
@@ -26,15 +26,32 @@
 import lombok.Builder;
 import lombok.Data;
 
+/**
+ * Subscription is an offering with one or more variants. Defines a specific metering model. Has a
+ * single technical fingerprint. Defines a set of metrics.
+ */
 @Data
 @Builder
 public class Subscription {
 
+  /**
+   * A family of solutions that is logically related, having one or more subscriptions distinguished
+   * by unique technical fingerprints (e.g. different arches)
+   */
   @NotNull @NotEmpty private String platform; // required
 
   @NotNull @NotEmpty private String id; // required
 
-  private List<String> requiredSubscriptions;
```

Comment:
This rename discussed in a slack thread.  Was easier to just do it here than to rehash it again later.